packages/inter-protocol/src/psm/psm.js

```diff
@@ -97,6 +98,20 @@ export const start = async (zcf, privateArgs, baggage) => {
   const feePool = provideEmptyZcfSeat('feePoolSeat');
   const stage = provideEmptyZcfSeat('stageSeat');
 
+  // Metrics
+  const { metricsPublication, metricsSubscription } = makeMetricsPublisherKit(
+    privateArgs.storageNode,
+    privateArgs.marshaller,
+  );
+  const updateMetrics = () => {
+    metricsPublication.updateState(
+      harden({
+        poolBalance: anchorPool.getAmountAllocated('Anchor', anchorBrand),
```

Comment:
That's all we need to recover the state, but the econ committee will also want to know about flows through the system.  We should track the amounts that passed through `giveStable` and `wantStable` as well.

Reply:
@rowgraus @dtribble @btulloh it would be good to get your input on this.